src/Tickets/Flexible_Tickets/Series_Passes/Reports.php

```diff
@@ -0,0 +1,229 @@
+<?php
+/**
+ * Handles the reports integrations for the Series Passes.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Flexible_Tickets;
+ */
+
+namespace TEC\Tickets\Flexible_Tickets\Series_Passes;
+
+use TEC\Events_Pro\Custom_Tables\V1\Series\Post_Type as Series_Post_Type;
+use TEC\Tickets\Commerce\Reports\Data\Order_Summary;
+use TEC\Tickets\Flexible_Tickets\Series_Passes;
+use TEC\Tickets\Flexible_Tickets\Templates\Admin_Views;
+use Tribe__Tickets__Tickets as Tickets;
+use Tribe__Tickets__Ticket_Object as Ticket_Object;
+
+/**
+ * Class Reports.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Flexible_Tickets;
+ */
+class Reports {
+
+	/**
+	 * A reference to the admin views.
+	 *
+	 * @since TBD
+	 *
+	 * @var Admin_Views
+	 */
+	private Admin_Views $admin_views;
+
+	public function __construct( Admin_Views $admin_views ) {
+		$this->admin_views = $admin_views;
+	}
+
+	/**
+	 * Registers the hooks.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function register_hooks(): void {
```

Comment:
I meant something else.
Any `add_action` or `add_filter` call should stay in the Controller, the `Base` one in this case.
The link I had provided in my comment showed the Controller delegating the actual method to a specialized class.

Reply:
As this was handling reporting related to Series Passes only (for now) so, I thought it belonged under Series pass namespace.